packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -59,27 +75,28 @@ const AdvertiserPage = () => {
             })}
         >
             <BlockUserModal
+                advertiser_name={name}
                 is_advertiser_blocked={!!advertiser_page_store.is_counterparty_advertiser_blocked}
                 is_block_user_modal_open={general_store.is_block_user_modal_open}
-                onCancel={() => general_store.setIsBlockUserModalOpen(false)}
-                onSubmit={() =>
-                    general_store.blockUnblockUser(
-                        !advertiser_page_store.is_counterparty_advertiser_blocked,
-                        advertiser_page_store.advertiser_details_id
-                    )
-                }
+                onCancel={onCancel}
+                onSubmit={onSubmit}
             />
             <BuySellModal
                 selected_ad={advertiser_page_store.advert}
                 should_show_popup={advertiser_page_store.show_ad_popup}
                 setShouldShowPopup={advertiser_page_store.setShowAdPopup}
                 table_type={advertiser_page_store.counterparty_type === buy_sell.BUY ? buy_sell.BUY : buy_sell.SELL}
             />
-            <PageReturn
-                className='buy-sell__advertiser-page-return'
-                onClick={buy_sell_store.hideAdvertiserPage}
-                page_title={localize("Advertiser's page")}
-            />
+            <div className='advertiser-page__page-return-header'>
```

Comment:
Nit, since this is a header class containing other components, it can just be named `header`. 😛 

```suggestion
            <div className='advertiser-page__header'>
```

Reply:
there's another __header, can I name it return-header?